python/ray/rllib/models/action_dist.py

```diff
@@ -28,11 +28,14 @@ class ActionDistribution(object):
 
     Args:
       inputs (Tensor): The input vector to compute samples from.
+      model_config (dict): Optional model config dict
+          (as defined in catalog.py)
     """
 
     @DeveloperAPI
-    def __init__(self, inputs):
+    def __init__(self, inputs, model_config=None):
```

Comment:
To avoid accidents where we forget to pass this, consider making it a required argument.

Reply:
I lean towards making it required but this may break other users' code.

Reply:
Hm since this is DeveloperAPI I would say it's ok to err on the side of avoiding bugs vs backwards compatibility.